src/lib/openapi-to-json-schema.ts

```diff
@@ -475,7 +491,7 @@ export default function toJSONSchema(
       }
     } else if (schema.type === 'object') {
       if ('properties' in schema) {
-        Object.keys(schema.properties).map(prop => {
+        Object.keys(schema.properties).forEach(prop => {
```

Comment:
Changed this to a `forEach` because I'm not reshaping `schema.properties` to anything with `map`.